lib/index.js

```diff
@@ -23,7 +25,15 @@ var Castle = module.exports = integration('Castle')
 Castle.prototype.initialize = function() {
   window._castle = window._castle || {};
   window._castle.q = window._castle.q || [];
-  window._castle.q.push(['setAppId', this.options.appId]);
+  window._castle.q.push(['setKey', this.options.publishableKey]);
+
+  if (this.options.cookieDomain) {
+    window._castle.q.push(['setCookieDomain', this.options.cookieDomain]);
+  }
+
+  if (this.options.autoPageview === false) {
+    window._castle.q.push(['autoTrack', this.options.autoPageview]);
```

Comment:
@wallin so when this happens, the library will start tracking pageviews automatically on its own?

The default behavior should be that autotracking is disabled so that customers can just explicitly tell you when there is a page view via our `.page()` call.

Some folks have pretty custom pageview tracking set up so probably better to just use segment to send page data explicitly by default